src/components/Footer.tsx

```diff
@@ -0,0 +1,79 @@
+import classNames from 'classnames';
+import { getActiveTodos, getCompletedTodos } from '../services/functions';
+import { Filter } from '../types/Filter';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  filterBy: Filter;
+  handleSelect: (filterBy: Filter) => void;
+  onDelete: (id: number) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filterBy,
+  handleSelect,
+  onDelete,
+}) => {
+  const clearCompleted = () => {
+    const allCompleted = getCompletedTodos(todos);
+
+    return allCompleted.map(todo => onDelete(todo.id));
```

Comment:
Should this function return something? If no let's just use forEach

And you are calling `getCompletedTodos` funtion 2 times in the component, let's just declare a variable for it out of this funtion

``